woocommerce/woocommerce-android
# Display gift cards in order details

Closes: #8399

### Why
We are adding read-only support for the Gift Cards extension

### Description
This PR adds the necessary code to display Gift Cards in the Order Details Screen. It does that by adding the OrderDetailGiftCardListView to the order detail fragment XML and the giftCards LiveData field to the OrderDetailViewModel.
Disclaimer: This code is not yet connected to any data source, so it will show a Gift Card section for every order, even if it was not paid with any gift card. This PR is just to set up the Gift Card section UI.

### Testing instructions
1. Open the Orders tab
2. Select any order
3. Check that the Gift Card section is shown
4. Check that the price gift card price is shown in the Payment section

### Images/gif
| Light mode | Dark mode |
| ------------- | ------------- |
| <img width="300" src="https://user-images.githubusercontent.com/18119390/228694703-37188ec3-5324-44fa-8cd1-29912851fd59.png" />  | <img width="300" src="https://user-images.githubusercontent.com/18119390/228694867-cfbad4f4-79fd-4fb4-a392-16439e50d689.png" />  |


https://user-images.githubusercontent.com/18119390/229206427-eee36529-b7e4-4796-9e5d-1f425b59a04b.mp4
